gensrc/thrift/FrontendService.thrift

```diff
@@ -510,6 +538,7 @@ struct TLoadTxnCommitRequest {
     7: required i64 txnId
     8: required bool sync
     9: optional list<Types.TTabletCommitInfo> commitInfos
+    10: optional TTxnStateExtra txnStateExtra
```

Comment:
This struct is only used when transaction commit, is `TTransactionCommitAttachment` better?